packages/react-ui/src/themes.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Theme type definitions for Phantom UI
+ * Simple theme focused on colors and border radius
+ */
+
+import { hexToRgba } from "./utils";
+
+export interface PhantomTheme {
+  // Background color for modal
+  background: string;
+
+  // Secondary color for text, borders, dividers
+  secondary: string;
+
+  // Error color
+  error: string;
+
+  // Success color
+  success: string;
+
+  // Primary text color
+  text: string;
+
+  // Overlay background (with opacity)
+  overlay: string;
+
+  // Border radius for buttons and modal
+  borderRadius: string;
+
+  // Brand color
+  brand: string;
+}
+
+export type PhantomThemeWithAux = PhantomTheme & {
+  aux: string; // Auxiliary color derived from secondary with opacity
+};
```

Comment:
We can increase the type safety with a hex type literal:
```suggestion
type HexColor = `#${string}`

export interface PhantomTheme {
  // Background color for modal
  background: HexColor;

  // Secondary color for text, borders, dividers
  secondary: HexColor;

  // Error color
  error: HexColor;

  // Success color
  success: HexColor;

  // Primary text color
  text: HexColor;

  // Overlay background (with opacity)
  overlay: HexColor;

  // Border radius for buttons and modal
  borderRadius: HexColor;

  // Brand color
  brand: HexColor;
}

export type PhantomThemeWithAux = PhantomTheme & {
  aux: HexColor; // Auxiliary color derived from secondary with opacity
};
```